packages/@okta/vuepress-site/docs/reference/api/brands/index.md

```diff
@@ -0,0 +1,1389 @@
+
+| Property                              | Type     | Description                                                                                                                | Default Value     |
+| ------------------------------------- | -------- | -------------------------------------------------------------------------------------------------------------------------- | ----------------- |
+| `primaryColorHex`                     | String   | Primary color hex code.                                                                                                    | `#1662dd`         |
+| `primaryColorContrastHex`             | String   | (Optional) Primary color contrast hex code. Accepted values: `#000000`, `#ffffff`.                                         | `#000000`         |
+| `secondaryColorHex`                   | String   | Secondary color hex code.                                                                                                  | `#ebebed`         |
+| `secondaryColorContrastHex`           | String   | (Optional) Secondary color contrast hex code. Accepted values: `#000000`, `#ffffff`.                                       | `#000000`         |
+| `signInPageTouchPointVariant`         | Enum     | Variant for Sign In page touch point. Accepted values: `OKTA_DEFAULT`, `BACKGROUND_SECONDARY_COLOR`, `BACKGROUND_IMAGE`.   | `OKTA_DEFAULT`    |
+| `endUserDashboardTouchPointVariant`   | Enum     | Variant for End user dashboard touch point. Accepted values: `OKTA_DEFAULT`, `WHITE_LOGO_BACKGROUND`, `FULL_THEME`.        | `OKTA_DEFAULT`    |
+| `errorPageTouchPointVariant`          | Enum     | Variant for Error page touch point. Accepted values: `OKTA_DEFAULT`, `BACKGROUND_SECONDARY_COLOR`, `BACKGROUND_IMAGE`.     | `OKTA_DEFAULT`    |
+| `emailTemplateTouchPointVariant`      | Enum     | Variant for Email templates touch point. Accepted values: `OKTA_DEFAULT`, `FULL_THEME`.                                    | `OKTA_DEFAULT`    |
+
+#### Theme Touch Point Variant Definitions
+
+##### Sign In Page Touch Point Variant:
+
+> **Note:** For non `OKTA_DEFAULT` variant, `primaryColorHex` will be used for buttons background color and `primaryColorContrastHex` will be used to identify shade for button text
+
+| Enum Value                      | Description                                                                                          |
+| ------------------------------- | ---------------------------------------------------------------------------------------------------- |
+| `OKTA_DEFAULT`                  | Use Okta default Logo with no background image along with Okta default colors in the sign in page.   |
+| `BACKGROUND_SECONDARY_COLOR`    | Use Logo from Theme with `secondaryColorHex` as background color for the sign in page.               |
+| `BACKGROUND_IMAGE`              | Use Logo and background image from Theme.                                                            |
```

Comment:
@TiffanyNg-okta please verify the touch point variant applicability

Reply:
Looks good for sign in page variants. Thanks Arsalan!

nit: We should probably be consistent with capitalization/casing as in `Logo` vs `background image/color` vs `sign in page`. Maybe that's something docs team can help with.

Reply:
thanks for verifying. 
good catch. I have used lower case for now to make the content consistent